ted_sws/notice_packager/model/metadata.py

```diff
@@ -35,38 +53,52 @@
 LANGUAGE = LANGUAGES[0]
 USES_LANGUAGE = "MUL"
 
-ACTION_CREATE = "create"
-ACTION_UPDATE = "update"
-ACCEPTED_ACTIONS = [ACTION_CREATE, ACTION_UPDATE]
-
 REVISION = "0"
 
 
-def validate_notice_action_type(v):
-    if v not in ACCEPTED_ACTIONS:
-        raise ValueError('No such action: %s' % v)
+def validate_mets_type(v):
```

Comment:
what is v ? victory ?

Reply:
it’s a short name for value when used in a very small context (likke: k => v: key => value)